packages/components/src/templates/teaser-media.stories.ts

```diff
@@ -0,0 +1,106 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Teaser Media',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+/**
+ *
+ * - Aligment: Teaser contents can be center aligned if desired.
+ * - Paddings: Can be changed as desired.
+ * - Headline size: Can be changed as desired.
+ *
+ * ```
+ * ```
+ */
+
+export const Default = {
+  render: () =>
+    html` <style>
+        #part-content sd-teaser-media::part(content) {
+          padding: 60px;
+        }
+      </style>
+      <div class="flex flex-wrap gap-8 w-[600px]">
+        <div id="part-content">
+          <sd-teaser-media variant="gradient-dark" class="min-w-[435px] max-w-4xl">
```

Comment:
The components are not responsive, I think it's because you are defining specific widths for the wrapper and the component itself.

Reply:
Updated